cmd/stat.go

```diff
@@ -59,8 +60,9 @@ func (stat statMessage) String() (msg string) {
 	// Format properly for alignment based on maxKey leng
 	stat.Key = fmt.Sprintf("%-10s: %s", "Name", stat.Key)
 	msgBuilder.WriteString(console.Colorize("Name", stat.Key) + "\n")
-	msgBuilder.WriteString(fmt.Sprintf("%-10s: %s ", "Date", stat.Date.Format(printDate)) + "\n")
-	msgBuilder.WriteString(fmt.Sprintf("%-10s: %-6s ", "Size", humanize.IBytes(uint64(stat.Size))) + "\n")
```

Comment:
Is removing size intended ? I do not think that we should do it. I think statMessage already contains a field for target type (file or dir), we can avoid printing the size if it is not a file

Reply:
fixed - intended to remove it only for folder